index.Rmd

```diff
@@ -89,7 +91,7 @@ roads_water_plot<-gplot(talbot_rf) +
   geom_raster(mapping=aes(fill=value)) +
   coord_cartesian() +
   xlab("longitude") +
-  ylab("latitude")
+  ylab("latitude") 
```

Comment:
I know you wanted to change the colors for this. Maybe you could do something like + scale_fill_gradient((low="green", high="blue"), so you would have land represented as green and water as blue still.